gradebookng/tool/src/webapp/scripts/gradebook-settings.js

```diff
@@ -100,6 +108,23 @@ GradebookGradingSchemaSettings.prototype.focusLastRow = function() {
   Wicket.Focus.setFocusOnId($input.attr("id"));
 }
 
+GradebookGradingSchemaSettings.prototype.getFocusedCell = function() {
+  var $activeElement = $(document.activeElement);
+  if ($activeElement.hasClass('schema-input')) {
+    sakai.gradebookng.settings.gradingschemas.cellName = $activeElement.attr('name');
+  }
+}
+
+GradebookGradingSchemaSettings.prototype.focusPreviousCell = function() {
+  // This is a trick to focus the previous focused cell after table re-render
+  var cellName = sakai.gradebookng.settings.gradingschemas.cellName;
+  var $inputSameName = $('[name="' + cellName + '"]');
+  if ($inputSameName.length > 0) {
+    $inputSameName.focus();
```

Comment:
Generally, I'd use vanillajs here. You can use querySelectorAll and forEach on the results of that.